src/test/java/lotto/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package lotto;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.ParameterizedTest.DEFAULT_DISPLAY_NAME;
+
+class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set - size 메서드 테스트")
+    @Test
+    void size() {
+        assertThat(numbers).hasSize(3);
+    }
+
+    @ParameterizedTest(name = "Set - contains 테스트 (Parameterized value) " + DEFAULT_DISPLAY_NAME)
+    @ValueSource(ints = { 1, 2, 3 })
+    void contains(int input) {
+        assertThat(numbers.contains(input)).isTrue();
+    }
```

Comment:
이런 메소드도 활용할 수도 있으니 참고만 해주세요!!

```suggestion
    void contains(int input) {
        assertThat(numbers).contains(input);
    }
```

Reply:
동일한 검증 로직을 단순화 할 수 있는 메서드가 존재한다면 활용하는 것이 바람직하다고 생각합니다
코멘트 감사합니다 !